certbot/display/ops.py

```diff
@@ -289,3 +289,49 @@ def _gen_https_names(domains):
             domains[-1])
 
     return ""
+
+
+def _get_validated(method, validator, message, default=None, **kwargs):
+    if default is not None:
+        try:
+            validator(default)
+        except errors.PluginError as error:
+            raise AssertionError("Invalid default {0}".format(default))
```

Comment:
nits:

1. We should log the raised exception out of `validator`. It probably isn't useful, but it might be so lets throw it in a log. Easiest way to do this is `logger.debug("some msg", exc_info=True)`.
2. This probably will never matter, but you can turn off assertions in Python by passing `-O` to the interpreter, but this doesn't work if you explicitly raise an `AssertionError`. Instead, we could do `assert False, "Invalid default {}".format(default)` here.